### PR TITLE
Kses - Part 8

### PR DESCRIPTION
**Pull request by hoseinrafiei**
This is related to https://github.com/eventespresso/event-espresso-core/issues/3738 issue.

Applied `wp_kses` function to most of variables in **/core** directory.

Here is the testing checklist: https://trello.com/c/wV6x95WL/1159-ee4-core-testing-checklist-pr-3806

### CODE REVIEW

**Comment by hoseinrafiei** [comment]
@tn3rb Please check this PR and let me know your thoughts.

File: core/db_classes/EE_CSV.class.php [hunk 1]
```diff
@@ -420,11 +420,11 @@ public function write_data_array_to_csv($filehandle, $data)
             if (EEH_Array::is_associative_array($item_in_top_level_array)) {
                 // its associative, so we want to output its keys as column headers
                 $keys = array_keys($item_in_top_level_array);
-                echo $this->fputcsv2($filehandle, $keys);
+                $this->fputcsv2($filehandle, $keys);
```
Comment (tn3rb) [inline core/db_classes/EE_CSV.class.php:423]: lolz 👍🏻

**Review by tn3rb — APPROVED** [review]
### Awesome 💯 

looks like smooth sailing now ⛵

**Comment by hoseinrafiei** [comment]
@mohdshakeel and @Saam01 

Please start testing this branch. Since we modified core libraries, please test everything in EE in both of caff and decaf modes. You can find the checklist in first PR.

**Comment by mohdshakeel** [comment]
@hoseinrafiei We have started the work on it.

**Comment by Saam01** [comment]
@hoseinrafiei We are working with the PR.

**Comment by hoseinrafiei** [comment]
Added a fix that closes https://github.com/eventespresso/event-espresso-core/issues/3810 issue.

**Comment by mohdshakeel** [comment]
> Added a fix for the #3810 issue.

Its working perfectly now.

**Comment by Saam01** [comment]
We've finished testing this PR.

**Pull request merged**